packages/beacon-node/src/metrics/metrics/lodestar.ts

```diff
@@ -110,6 +110,11 @@ export function createLodestarMetrics(
         help: "Current count of pending items in reqRespBridgeReqCaller data structure",
       }),
     },
+    networkWorkerWireEventsOnMainThreadLatency: register.histogram({
```

Comment:
it'd be great to have suffix in the name representing unit, like 
```suggestion
    networkWorkerWireEventsOnMainThreadLatencySec: register.histogram({
```

Reply:
Updated.

Reply:
aren't all metrics in seconds?

Reply:
yes.  Dapplion had me change it to be in ISU units

Reply:
> aren't all metrics in seconds?

@nflaig point is that since all metrics are seconds it may not be able to specify so in the name. Actually we don't do that anywhere else (please check if this is true). So for consistency we should: use seconds everywhere, and don't declare that on the name. For code variables we use multiple units so in that case it makes sense to specify the unit

Reply:
The only metrics I found that are suffixed with `Seconds` are in `packages/beacon-node/src/metrics/metrics/lodestar.ts`.

E.g. `prevEpochAttestationsMinDelaySeconds` 
https://github.com/ChainSafe/lodestar/blob/201dfc805e7f7a60c9b03474d6c96ba452ca7094/packages/beacon-node/src/metrics/metrics/lodestar.ts#L799

Most metrics don't have this suffix, a more common suffix is `Time` for time based metrics
https://github.com/ChainSafe/lodestar/blob/201dfc805e7f7a60c9b03474d6c96ba452ca7094/packages/beacon-node/src/metrics/metrics/lodestar.ts#L1398